src/icons/Icons.stories.js

```diff
@@ -1,5 +1,5 @@
 import React from "react";
-import IconSelection from "dist/icons/selection.json";
+import iconSelection from "dist/icons/selection.json";
```

Comment:
nitpick for naming consistency

Reply:
is the first letter lowercase convention here because `iconSelection` is called as a function and not a component?

Reply:
yeah, PascalCase should be reserved for constructors or components